document/FileFormat.mdk

```diff
@@ -2385,7 +2390,7 @@ containing a message relevant to this call stack.
 
 A `stack` object shall contain a property named `frames` whose value is an array of one or more `stackFrame` objects (&sect;[#stackFrame]).
 This array shall include every function call in the stack for which the tool has information,
-and the entries that are present shall occur in chronological order with the most recent (innermost) call first and the least recent (outermost) call last.
+and the entries that are present shall occur in nesting order with the most recent (innermost) call first and the least recent (outermost) call last.
```

Comment:
I don't understand this change and it doesn't clarify anything. what is 'nesting' order? previously, the statement was clear